docs/articles/documentation/using-testcafe/common-concepts/using-browsers.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: docs
+title: Using Browsers
+permalink: /documentation/using-testcafe/common-concepts/using-browsers.html
+---
+# Using Browsers
+
+TestCafe comes with a [built-in library](#built-in-library) and several [plugins](#browser-provider-plugins) that allow you to easily set up testing in different web browsers.
+
+## Built-in Library
+
+The [testcafe-browser-natives](https://github.com/DevExpress/testcafe-browser-natives) built-in library is used to work with popular local browsers during testing: to open and close them, to take screenshots of browser windows, etc.
+To run tests in these browsers, you just need to specify them, and TestCafe will perform all required actions on browser windows by itself.
+
+You can select browsers in one of two ways:
+
+* through the command line (see [Local Browsers](../command-line-interface.md#local-browsers))
+* programmatically (by using the [runner.browsers](../programming-interface/runner.md#browsers) method)
+
+### Browser Names
+
+To identify browsers, browser short names are used. They allow you to avoid dealing with paths to browser executables or other awkward ways to refer to a browser.
+
+The following table lists all available browser names.
+
+Browser           | Alias
+----------------- | ----------
+Chromium          | `chromium`
+Google Chrome     | `chrome`
+Internet Explorer | `ie`
+Microsoft Edge    | `edge`
+Mozilla Firefox   | `firefox`
+Opera             | `opera`
+Safari            | `safari`
+
+To obtain the list of browsers installed on the current machine, run TestCafe
+from the command line with the [-b option](../command-line-interface.md#-b---list-browsers).
+
+## Browser Provider Plugins
+
+TestCafe provides plugins for testing in [Sauce Labs](https://saucelabs.com/) and [PhantomJS](http://phantomjs.org/) browsers:
+
+* [testcafe-browser-provider-saucelabs](https://github.com/DevExpress/testcafe-browser-provider-saucelabs)
+* [testcafe-browser-provider-phantomjs](https://github.com/DevExpress/testcafe-browser-provider-phantomjs)
+
+You can also create a [custom browser provider plugin](../../extending-testcafe/custom-browser-provider-plugin/) that will fulfill your needs.
+
+For more information about browser provider plugins, see the following sections.
+
+* [Searching for Browser Provider Plugins](#searching-for-browser-provider-plugins)
+* [Installing the Browser Providers](#installing-the-browser-providers)
+* [Using the Browser Providers](#using-the-browser-providers)
+
+### Searching for Browser Provider Plugins
+
+Browser provider plugins are npm packages. The package name consists of two parts - the `{testcafe-browser-provider-}` prefix and the name of a provider itself; for example, `{testcafe-browser-provider-saucelabs}`.
```

Comment:
Here and in other places: why we put prefix and name in braces? I will definetly not find anything by `{testcafe-browser-provider-saucelabs}` query. Also, it's wrong to say that I should use `{testcafe-browser-provider-}` prefix